Packs/VirusTotal/Integrations/VirusTotal_V3_Premium/VirusTotal_V3_Premium.py

```diff
@@ -0,0 +1,1028 @@
+        outputs['data'] = decrease_data_size(outputs.get('data', []))
+    return CommandResults(
+        f'{INTEGRATION_ENTRY_CONTEXT}.LiveHuntFiles',
+        'id',
+        readable_output=tableToMarkdown(
+            'Notifications file listed:',
+            [
+                {**item.get('attributes', {}), 'id': item.get('id')} for item in raw_response.get('data', [])
+            ],
+            headers=['id', 'meaningful_name', 'last_analysis_stats']
+        ),
+        outputs=outputs,
+        raw_response=raw_response
+    )
+
+
+def list_files_by_rule(client: Client, args: dict) -> CommandResults:
+    """Get a VT Hunting Livehunt ruleset by hunting notification files relationship."""
+    id_ = args['id']
+    limit = arg_to_number(args.get('limit'))
+    cursor = args.get(args.get('cursor'))
+    raw_response = client.list_files_by_rule(id_, cursor, limit)
+    data = raw_response.get('data', [])
+    return CommandResults(
+        f'{INTEGRATION_ENTRY_CONTEXT}.LiveHuntFiles',
+        'id',
+        outputs=data,
+        raw_response=raw_response
+    )
+
+
+def list_retrohunt_jobs(client: Client, args: dict) -> CommandResults:
+    """Retrieve retrohunt jobs"""
+    limit = arg_to_number(args.get('limit'))
+    cursor = args.get('cursor')
+    filter_ = args.get('filter')
+    raw_response = client.list_retrohunt_jobs(filter_, cursor, limit)
+    data = raw_response.get('data', [])
+    return CommandResults(
+        f'{INTEGRATION_ENTRY_CONTEXT}.RetroHuntJob',
+        'id',
+        readable_output=tableToMarkdown(
+            'Retrohunt jobs listed:',
+            [
+                {
+                    **item.get('attributes', {}),
+                    'id': item.get('id')
+                } for item in data
+            ],
+            headers=['id', 'corpus', 'status', 'rules']
+        ),
+        outputs=data,
+        raw_response=raw_response
+    )
+
+
+def create_retrohunt_jobs(client: Client, args: dict) -> CommandResults:
+    """Create a new retrohunt job."""
+    rules = stringUnEscape(args.get('rules'))
+    notification_email = argToList(args.get('notification_email'))
+    corpus = args.get('corpus')  # main / goodware
+    start_time, end_time = args.get('start_time'), args.get('end_time')
+    time_range = get_time_range_object(start_time, end_time)
+    raw_response = client.create_retrohunt_job(rules, notification_email, corpus, time_range)
+    data = raw_response.get('data', {})
+    return CommandResults(
+        f'{INTEGRATION_ENTRY_CONTEXT}.RetroHuntJob',
+        'id',
+        readable_output=tableToMarkdown(
+            'Retrohunt job has been successfully created!',
+            {
+                **data,
+                **data.get('attributes', {})
+            },
+            headers=['id', 'corpus', 'status', 'rules']
+        ),
+        outputs=data,
+        raw_response=raw_response
+    )
+
+
+def get_retrohunt_job_by_id(client: Client, args: dict) -> CommandResults:
```

Comment:
please convert timestamps to human readable dates
![image](https://user-images.githubusercontent.com/31018228/111455068-07872c80-871e-11eb-8dcc-36e6d92b1d34.png)